src/api/files.rs

```diff
@@ -122,6 +122,15 @@ where
             )
             .await
     }
+
+    pub async fn files_delete(
```

Comment:
Nit: can you put please a comment with HTTP url to the API docs for this method as we have for others please? :)